lib/src/text-input/TextInput.test.js

```diff
@@ -394,7 +389,6 @@ describe("TextInput component tests", () => {
     expect(options[0].getAttribute("aria-selected")).toBeNull();
   });
 });
-
 describe("TextInput component synchronous autosuggest tests", () => {
   test("Autosuggest is displayed when the input gains focus", async () => {
```

Comment:
Remove the `async` word from this test and the two next to it, since it makes no sense to have it (there isn't any `await` inside of them).

Reply:
Done